recipes/toolchain/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: toolchain
```

Comment:
> Got it, still learning the wily ways of conda.

That's a pretty new addition though so I'm not sure how many people are aware of it.

> Is there a canonical way we can start capturing and documenting best practices?

We have been trying to document some things like this in the [wiki]( https://github.com/conda-forge/staged-recipes/wiki ). Though it certainly could use some more love. Right now it is mainly Windows tricks, pinning, and a few links.

Also, we have [guidelines]( https://github.com/conda-forge/conda-forge.github.io/blob/2a508a065c3316c42df01775553c40d2e21baa14/docs/guidelines.md ) for additions. We are working on a PR template for staged-recipes ( https://github.com/conda-forge/staged-recipes/pull/550 ).

One of the big things we have been trying to do of late is absorb some of the repetitive practices into a package like this one or this other one for Windows ( https://github.com/conda-forge/staged-recipes/pull/617 ). There is also an ongoing discussion in this issue ( https://github.com/conda-forge/staged-recipes/issues/535 ) about absorbing boilerplate in different ways.

> I find the conda documentation lacking (I mean, obviously one could submit conda PRs, but I'm talking about more a cookbook for common scenarios).

That doesn't hurt either. The conda docs live in this [repo]( https://github.com/conda/conda-docs ). There is also a conda [wiki]( https://github.com/conda/conda/wiki ), which can be nice. Also, conda-build has its own [wiki]( https://github.com/conda/conda-build/wiki ). Though that seems mainly to have ideas about Windows.